addon/mixins/copyable.js

```diff
@@ -192,9 +192,9 @@ export default Ember.Mixin.create({
                   Supports Ember Data 2.5+
             */
           if (meta.kind === 'hasMany') {
-            copyRef.hasManyRelationship.addRecords(ref.hasManyRelationship.members);
+            copyRef.hasManyRelationship.addInternalModels(ref.hasManyRelationship.members);
```

Comment:
Can we check if `addInternalModels` exists and if so, using it instead of `addRecords`? That way this change is backwards compatible.